### PR TITLE
Expose the ion endpoint as a parameter

### PR DESCRIPTION
**Pull request by xuelongmu**
Follow up to https://github.com/CesiumGS/cesium-native/pull/444, this PR exposes the cesium-native ion endpoint parameter in Cesium for Unreal.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @xuelongmu!

* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium-unreal/blob/expose-ion-endpoint-unreal/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.

Reviewers, don't forget to make sure that:

- [ ] [Cesium for Unreal Samples](https://github.com/CesiumGS/cesium-unreal-samples) works.

File: Source/CesiumRuntime/Public/Cesium3DTileset.h [hunk 1]
```diff
@@ -604,6 +617,12 @@ class CESIUMRUNTIME_API ACesium3DTileset : public AActor {
   UFUNCTION(BlueprintSetter, Category = "Cesium")
   void SetIonAccessToken(FString InAccessToken);
 
+  UFUNCTION(BlueprintGetter, Category = "Cesium")
+  FString GetIonAssetEndpointUrl() const { return IonAssetEndpointUrl; }
+
+  UFUNCTION(BlueprintSetter, Category = "Cesium")
+  void SetIonAssetEndpointUrl(FString InIonAssetEndpointUrl);
```
[the remark this answers is not in the document]
Comment (xuelongmu) [reply Source/CesiumRuntime/Public/Cesium3DTileset.h]: Thanks, corrected the others as well. For blueprint pins, the UPARAM(ref) is only needed for input pin variables that are not const, so for a `const FString&` it'll get correctly parsed as an input.
https://michaeljcole.github.io/wiki.unrealengine.com/UPARAM/

**Comment by xuelongmu** [comment]
Ready for another review, thanks!

**Comment by nithinp7** [comment]
Thanks @xuelongmu, I will merge this as soon as CI passes.

**Comment by nithinp7** [comment]
This is only failing on IOS with a known (and strange) problem we are looking at. I'm going to merge this, since we'll need to fix that issue before the next release anyways. 
cc: @kring

**Pull request merged**